packages/account-fs/agent/runtime.js

```diff
@@ -65,15 +70,31 @@ export class Runtime {
       case BROWSER_RUNTIME:
         return await localforage.getItem(key)
       case SERVER_RUNTIME:
-        if (key == SHOVEL_FS_ACCESS_KEY) {
-          // convert string to Uint8array
-          return uint8arrays.fromString(this.config[key], 'base64url')
-        } else if (key == SHOVEL_FS_FOREST_CID) {
-          //convert string to Uinst8array
-          return CID.parse(this.config[key]).bytes
+        if(this.redisClient) {
+          const fullKey = `agent:${this.config[SHOVEL_AGENT_DID]}:${key}`
+          let value = await this.redisClient.get(fullKey);
+          if (key == SHOVEL_FS_ACCESS_KEY) {
+            // convert string to Uint8array
+            return uint8arrays.fromString(value, 'base64url')
+          } else if (key == SHOVEL_FS_FOREST_CID) {
+            //convert string to Uinst8array
+            return CID.parse(value).bytes
+          }
+          else {
+            return value
+          }
         }
         else {
```

Comment:
this `else` would never trigger, if we are going to always use redis with `SERVER_RUNTIME`